adapter/internal/adapter/adapter.go

```diff
@@ -221,13 +221,10 @@ func Run(conf *config.Config) {
 		go synchronizer.UpdateRevokedTokens()
 		// Fetch Key Managers from APIM
 		synchronizer.FetchKeyManagersOnStartUp(conf)
-	}
-
-	throttlingEnabled := conf.Enforcer.Throttling.EnableGlobalEventPublishing
-	if throttlingEnabled {
```

Comment:
Is there a reason to remove this check

Reply:
this is adapter side check. here we're checking if throttling is enabled at enforcer config to fetch throttle data to adapter from CP. Ideally we should fetch eventhub data to adapter if eventhub is enabled and enforcers can subscribe to those data via xds if throttling is enabled